src/views.rs

```diff
@@ -161,6 +161,12 @@ pub struct OwnedCrate {
 pub struct EncodableMe {
     pub user: EncodablePrivateUser,
     pub owned_crates: Vec<OwnedCrate>,
+    pub meta: EncodableMeMeta,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, Copy)]
+pub struct EncodableMeMeta {
+    pub token_count: i64,
```

Comment:
thanks for asking this question. when I was working on this it seemed obvious to me, but now I'm not so sure anymore either. 😅 

I'm now wondering if it would be better to instead include the list of token IDs, like in a proper relationship declaration, but that would probably slightly slow down the query and for what we're trying to build it doesn't make a difference.

Maybe my confusion comes from using `user` for both other (GitHub) users, but also the logged in user account, but with exta information in the latter case.

Reply:
I thought about this a little bit more. I think my reason for putting this in `meta` was that this is derived data, which corresponds to the number of tokens that will be returned from the `GET /api/v1/me/tokens` request. It is not actually a property on the `user` resource, but rather the number of `token` relationships that this user has.

> Should `owned_crates` be in `EncodableMeMeta`, for example?

I guess it would be best if this was encoded as a proper relationship instead